akka-bbb-apps/src/main/scala/org/bigbluebutton/core/models/Users2x.scala

```diff
@@ -71,8 +71,12 @@ object Users2x {
     users.toVector.filter(u => !u.presenter)
   }
 
-  def findNotPresentersNorModerators(users: Users2x): Vector[UserState] = {
-    users.toVector.filter(u => !u.presenter && u.role != Roles.MODERATOR_ROLE)
+  def findNotPresentersNorModerators(users: Users2x, reduceDup: Boolean): Vector[UserState] = {

```

Comment:
Since with this change this method will only be used for the Random User functionality, it is justified to rename it to getRandomlyPickableUsers